distuv/exponential.go

```diff
@@ -199,13 +200,13 @@ func (e Exponential) StdDev() float64 {
 // of the samples.
 //
 // If weights is nil, the weights are assumed to be 1, otherwise panics if
-// len(samples) != len(weights). Panics if len(suffStat) != 1.
+// len(samples) != len(weights). Panics if len(suffStat) != NumSuffStat().
 func (Exponential) SuffStat(samples, weights, suffStat []float64) (nSamples float64) {
```

Comment:
Yeah. I think, in line with @btracey's request that I hold off on the factorisation matrix extraction revisions, that should probably wait until the monorepoisation has happened.

There should probably be an issue in gonum/gonum for that to aggregate issues https://github.com/gonum/gonum/issues/4.